app/controllers/api/v3/logins_controller.rb

```diff
@@ -21,17 +21,25 @@ def login_user
       render json: response, status: :ok
     else
       log_failure(result)
-      response = {
-        errors: {
-          user: [result.error_message]
-        }
-      }
-      render json: response, status: :unauthorized
+      render json: get_error_response(result), status: :unauthorized
     end
   end
 
   private
 
+  def get_error_response(result)
```

Comment:
I would name this `error_response`, the `get` is superfluous.

Reply:
okay